t/t2717-python-cli-plugins.t

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+test_description='Test command line plugin interface'
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 4 job
+
+export FLUX_CLI_PLUGINPATH=${FLUX_BUILD_DIR}/t/cli-plugins/cli/plugins
```

Comment:
This line needs to be moved above `test_under_flux` or else the job validator will not have `FLUX_CLI_PLUGINPATH` in its environment.

However, after fixing that every test is failing because the `CustomFluxionPolicyPlugin` always raises the error `hi hobbs`